pkg/tsdb/prometheus/querydata/exemplar/framer.go

```diff
@@ -64,8 +64,14 @@ func (f *Framer) Frames() (data.Frames, error) {
 		timeField.Append(b.Timestamp)
 		valueField.Append(b.Value)
 		for i, labelName := range labelNames {
-			labelValue, ok := b.Labels[labelName]
-			if !ok {
+			var labelValue string
+			for _, f := range b.Fields {
```

Comment:
Could you please use `bf` instead of `f`. `f` is being used already.

Reply:
Done.